template/App.tsx

```diff
@@ -56,43 +56,56 @@ function Section({children, title}: SectionProps): React.JSX.Element {
 }
 
 function App(): React.JSX.Element {
+  return (
+    <SafeAreaProvider>
+      <MainContent />
+    </SafeAreaProvider>
+  );
+}
```

Comment:
I understand the reason for this, but this will show a huge braking change block in users code that is not really required. Can't we just replace the `<SafeAreaView>` with `<SafeArewProvider>`?